contrib/opencollective_hookshot.js

```diff
@@ -0,0 +1,75 @@
+// Input is data object which is hook body
+// Output is result variable of form
+// {
+//     "version": "v2" // The version of the schema being returned from the function. This is always "v2".
+//     "empty": true|false, // Should the webhook be ignored and no output returned. The default is false (plain must be provided).
+//     "plain": "Some text", // The plaintext value to be used for the Matrix message.
+//     "html": "<b>Some</b> text", // The HTML value to be used for the Matrix message. If not provided, plain will be interpreted as markdown.
+//     "msgtype": "some.type", // The message type, such as m.notice or m.text, to be used for the Matrix message. If not provided, m.notice will be used.
+// }
+
+// Configure this for the collective you are receiving webhooks from.
+// Expense payloads (unlike transactions) don't have this in the data.
+const defaultCollective = {
+    slug: "sunpy",
+    name: "SunPy"
```

Comment:
Is this left from copy pasting one of their scripts?
Do we actually have a connection to them?

Reply:
This the collective I am using this transform function for, this is config for users. I can make that clearer.